source/more_sampling_tools.Rmd

```diff
@@ -652,8 +663,8 @@ n_ten_both
 We can answer the same question for *all* the trials, in the same way:
 
 ```{python}
-first_gold = first_trunks == 'Gold'
-second_gold = second_trunks == 'Gold'
+first_gold = (first_trunks == 'Gold')
```

Comment:
Here the parentheses are difficult to justify - no?   I wouldn't use them myself in this case.

Reply:
I always use them like this, and this is what's been recommended by Robert Kern on the SciPy list too as far as I recall. It makes it very clear that the thing you are dealing with here is... a thing.